### PR TITLE
move llm_bench and wwb to tools

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
   pull_request_target:
     types: [opened, edited, synchronize]
     paths:
-      - llm_bench/python/**
+      - tools/llm_bench/python/**
```
Comment (eaidova) [inline .github/workflows/labeler.yml:8]:
```suggestion
      - tools/llm_bench/**
```

File: llm_bench/python/who_what_bench/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Simple Accuracy Benchmark for Generative AI models
+
+> [!IMPORTANT]  
+> Who What Benchmark code was moved to [tools](../../tools/who_what_benchmark/) directory. Please navigate to the new directory for continue of tool usage.
```
Comment (AlexKoff88) [inline llm_bench/python/who_what_bench/README.md:4]:
```suggestion
> Who What Benchmark code was moved to [tools](../../../tools/who_what_benchmark/) directory. Please navigate to the new directory for continue of tool usage.
```

**Review by AlexKoff88 — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by AlexKoff88** [comment]
@alexsu52

**Comment by alexsu52** [comment]
> @alexsu52

NNCF will migrate after merging this PR.

**Comment by peterchen-intel** [comment]
Any plan to have C++ tool? If yes, shall we still use llm_bench/python folder?

**Comment by ilya-lavrenov** [comment]
> Any plan to have CPP llm_bench? If yes, shall we still use llm_bench/python folder?

I suppose no

**Comment by AlexKoff88** [comment]
Can we merge this? as it blocks the development a bit.

**Pull request merged**